build.sbt

```diff
@@ -17,6 +17,7 @@ val coreDependencies = Seq(
 )
 
 val testkitDependencies = Seq(
+  "com.typesafe.akka" %% "akka-testkit" % akkaVersion,
```

Comment:
Why did you add it to testkit and not only benchmarks?

Reply:
On the Jenkins node testkit did not compile with missing classes from the testkit dependency. Note that alpakka-kafka testkit already depended transitively on akka-testkit because of akka-stream-testkit. So this only adds transitive dependency explicitly.